ci/gpu/build.sh

```diff
@@ -89,7 +89,7 @@ gpuci_mamba_retry install -y \
                   "ucx-py=${UCX_PY_VERSION}"
 
 # https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_mamba_retry remove --force rapids-build-env rapids-notebook-env
+# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
```

Comment:
It was/has always been erroneous. Ops (@ajschmidt8) had recommended to change it to `gpuci_conda_retry` otherwise the operation does not work so thought I would go ahead and change it for future users to prevent them going through the same pain I did =)